### ISSUE TITLE
[QUESTION] add or delete receiver(s) dynamically

### ISSUE THREAD

**Issue by AlexandreLicinio** [sleinen/samplicator#70]
Is it possible to add or delete a receiver while the process is running ?
In case we use samplicator to copies udp from transport stream (video+audio) the outputs aren't evolves during the day, it could be nice to add or delete one or many receivers.
thanks!

**Comment by Yongyut12** [comment]
http://<git provider>/<username>/<repository>.git

**Comment by doup123** [comment]
@AlexandreLicinio did you you find a workaround for that?

**Comment by AlexandreLicinio** [comment]
@doup123 yes we moved to gstreamer with pipes or multicast if the kernel supports it without dropped packets
